api/HTMLFormControlsCollection.json

```diff
@@ -48,6 +48,54 @@
           "standard_track": true,
           "deprecated": false
         }
+      },
+      "namedItem": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLFormControlsCollection/namedItem",
+          "support": {
+            "webview_android": {
+              "version_added": true
+            },
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": false
+            },
+            "edge_mobile": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": "27"
```

Comment:
Actually, similar `notes` exist on the parent feature:

```
"firefox": {
  "version_added": "27",
  "notes": "Until Firefox 32 included, the eventual <a href='https://developer.mozilla.org/docs/Web/API/RadioNodeList'>RadioNodeList</a> returned by <code>namedItem</code> was a <a href='https://developer.mozilla.org/docs/Web/API/NodeList'>NodeList</a>."
},
"firefox_android": {
  "version_added": "27",
  "notes": "Until Firefox 32 included, the eventual <a href='https://developer.mozilla.org/docs/Web/API/RadioNodeList'>RadioNodeList</a> returned by <code>namedItem</code> was a <a href='https://developer.mozilla.org/docs/Web/API/NodeList'>NodeList</a>."
},
```

But you're right, we should probably move it here, shouldn't we?

Reply:
@ddbeck Here's what I came up with now:

```
"firefox": [
  {
    "version_added": "33"
  },
  {
    "version_added": "27",
    "version_removed": "33",
    "partial_implementation": true,
    "notes": "Returned a <a href='https://developer.mozilla.org/docs/Web/API/NodeList'><code>NodeList</code></a> instead of a <a href='https://developer.mozilla.org/docs/Web/API/RadioNodeList'><code>RadioNodeList</code></a>."
  }
],
"firefox_android": [
  {
    "version_added": "33"
  },
  {
    "version_added": "27",
    "version_removed": "33",
    "partial_implementation": true,
    "notes": "Returned a <a href='https://developer.mozilla.org/docs/Web/API/NodeList'><code>NodeList</code></a> instead of a <a href='https://developer.mozilla.org/docs/Web/API/RadioNodeList'><code>RadioNodeList</code></a>."
  }
],
```